Articles/WhatIs/WhatAreConferencesAndWorkshops.md

```diff
@@ -0,0 +1,28 @@
+### What are Conferences and Workshops for Better Scientific Software?
+#### Contributed by  [Rinku Gupta](https://github.com/rinkug)
+#### Publication date: April 20, 2023
+
+<!--- deck start --->
+Empowering researchers in scientifivc computing with knowledge-sharing, collaboration, and networking opportunities through conferences and workshops.
+<!--- deck end --->
+
+<!--- body start --->
+
+Conferences and workshops play a pivotal role in advancing scientific research in the field of scientific computing. These events offer researchers a valuable platform to showcase their latest findings, discuss and exchange ideas with other experts, and cultivate collaborations that drive innovation in their field. Beyond knowledge-sharing and collaboration, attending conferences and workshops enables researchers to remain up-to-date on the latest trends and advancements in their field, enabling them to refine their research approach and stay ahead of the curve.
+
+Conferences and workshops provide researchers with a unique opportunity to present their work to an audience of peers, fostering critical discussions that encourage the exchange of ideas and feedback. Through these platforms, researchers receive constructive criticism that enables them to fine-tune their ideas and approaches, resulting in the development of new insights, techniques, and approaches that can further advance their research agenda. Scientific computing software stacks are built for various applications from different domains. Conferences and workshops offer researchers a chance to connect with experts from other domains and fields, fostering interdisciplinary collaboration and knowledge-sharing. These events also provide excellent opportunities for industry, academia, national laboratories, and other research organizations to connect and exchange notes. Several of these factors result in awareness of new funding and grant opportunities. With both formal and informal settings, conferences/workshops are considered good environments to build deeper, meaningful relationships with one's colleagues and advance in the field.
+
+Networking and career advancement opportunities are other significant outcomes of attending conferences and workshops. Employers in the scientific computing field often use these events to recruit talented interns/employees who possess the drive, enthusiasm, skills, knowledge, or experience to help them achieve their goals. These venues provide an opportunity for all these organizations to promote and highlight their mission, values, and the projects they are currently working on. For younger researchers, conferences/workshops provide a platform to showcase their work, whether it's through presenting a poster, giving a talk, or participating in a panel. This can help them gain recognition for their research and raise their profile in their field.
```

Comment:
I guess I see a lot of repeat of some of the same basic ideas from previous paragraphs here. 

I think all of these points are great points to make.

I just think there is a far less wordy way of making them. And, it was my understanding that part of the goal of this particular style (e.g. what-is docs) of documents was to be short and sweet.

Reply:
I can try to take a whack later today if you want me too.

Reply:
Pleasee do!!!